RecoLocalTracker/SiPixelDigiReProducers/python/siPixelDigiMorphed_cfi.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
sorry to be pedantic, the name of the file still does not match the name of the module (less the `_cfi.py` suffix):
`siPixelDigisMorphed` vs `siPixelDigiMorphed_cfi.py` is missing `s` in `Digis`